tests/BookStore/Integration/Doctrine/DoctrineBookRepositoryTest.php

```diff
@@ -75,6 +76,8 @@ public function testOfId(): void
         $book = DummyBookFactory::createBook();
         $repository->save($book);
 
+        static::getContainer()->get(EntityManagerInterface::class)->clear();
+
         static::assertSame($book, $repository->ofId($book->id));
```

Comment:
```suggestion
        static::assertEquals($book, $repository->ofId($book->id));
```
As they're aren't the same reference anymore